### PR TITLE
Doc(how-to): Update documentation

### PR DESCRIPTION
**Pull request by ClausHolbechArista**
## Change Summary

Update documentation

## Component(s) name

`arista.avd`

## Proposed changes

- [x] Update installation documentation
- [x] Update release-notes to reflect new requirements and constraints

## How to test

N/A

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by carlbuchmann — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/docs/installation/requirements.md [hunk 1]
```diff
@@ -19,43 +19,26 @@ If you leverage [Cloudvision](https://www.arista.com/en/products/eos/eos-cloudvi
...
-### Python requirements installation
-
-In a shell, run the following command:
-
-```shell
-$ pip3 install -r ansible_collections/arista/avd/requirements.txt
-```
-
 ```pip
 --8<--
```
Comment (tgodaA) [inline ansible_collections/arista/avd/docs/installation/requirements.md:23]: May I know if this was a typo or it has some significance?

**Review by tgodaA — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/docs/installation/collection-installation.md [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
 __arista.avd__ collection is available on [Ansible Galaxy](https://galaxy.ansible.com/arista/avd) server and can be automatically installed on your system.
 
+Make sure to install [Python requirements](requirements.md#additional-python-libraries-required) afterwards.
+
 ### Latest version
 
 ```shell
 $ ansible-galaxy collection install arista.avd
 ```
+!!! Warning: If you have an `ansible.cfg` file in the directory where you run `ansible-galaxy`, it may affect the directory under which the collection and dependencies will be installed.
```
Comment (titom73) [inline ansible_collections/arista/avd/docs/installation/collection-installation.md]: Syntax for mkdoc admonition is not like that, you should update it with:

```
!!! warning "your optional title to replace warning"
	lorem ipsum ....
```

(https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title)

File: ansible_collections/arista/avd/docs/installation/requirements.md (hunk 1 above)
Comment (titom73) [reply ansible_collections/arista/avd/docs/installation/requirements.md:23]: It is syntax to include content of a file, so in this situation, there is no need to update this section each time we change requirements.txt file.

**Review by titom73 — CHANGES_REQUESTED** [review]

File: ansible_collections/arista/avd/docs/installation/collection-installation.md (hunk 2 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/docs/installation/collection-installation.md]: Fixed, same for note in the same file.

**Pull request merged**